great-idea/css/index.css

```diff
@@ -203,30 +214,33 @@ nav a {
 address h2 {
 	margin: 20px;
 }
-
-footer p {
-	margin: 30px 0;
-	text-align: center;
-}
-
-
-
 .services-header-img{ 
     display: flex;
     justify-content: center;
 	width: 100%;
     justify-content: center;
     margin: 0 0 30px 0;
 }
+footer p {
+	margin: 30px 0;
+	text-align: center;
+}
+
+
+.servText{
+	font-size: 15px;
+}
+
 
 
 .boxes{
 	display: flex;
 	flex-wrap: wrap;
 	flex-direction: column;
+	border-top: 1px solid black;
 	border-bottom: 3px solid black;
 	margin: 30px 0 60px 0;
-	padding: 0 0 20px 0; 
+	padding: 30px 0 20px 0; 
```

Comment:
Getting used to doing all of the padding in one line like this is great. Keep up the good work!